site/website/blog/2021-02-02-jet-44-is-released.md

```diff
@@ -0,0 +1,239 @@
++--------------------+--------------------+
+|                   0|                   0|
+|                   1|                   2|
+|                   2|                   4|
+|                   3|                   6|
++--------------------+--------------------+
+4 row(s) selected
+```
+
+Here are two more examples with streaming SQL. Streaming queries never
+complete, so use `Ctrl+C` to cancel them after a while:
+
+```sql
+sql〉 SELECT * FROM TABLE(generate_stream(10));
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                   1|
+|                   2|
+^C
+Query cancelled.
+sql〉 SELECT * FROM TABLE(generate_stream(100)) WHERE v / 10 * 10 = v;
++--------------------+
+|                   v|
++--------------------+
+|                   0|
+|                  10|
+|                  20|
+^C
+Query cancelled.
+sql〉
+```
+
+For more examples with CSV files, Kafka and IMap, go to the
+[docs](/docs/sql/intro).
+
+We're currently very focused on bringing more features to our SQL, so
+stay tuned!
+
+## File Connector
+
+The [Unified File Connector
+API](https://jet-start.sh/docs/api/sources-sinks) gives you a simple way
+to read files, unified across different storage systems. Using the same
+API you can read files from the local filesystem, HDFS and its cloud
+derivatives such as Amazon S3, Google Cloud Storage or Azure Blob
+Storage. At the same time, the connector supports a variety of encoding
+formats - text files, CSV, JSON, Avro, etc., equally for all the storage
+systems.
+
+Here's how the Java syntax looks:
+
+```java
+BatchSource<String> source = FileSources
+    .files("/path/to/my/directory")
+    .build();
+```
+
+You specify the storage system type with the URI schema, for example to
+access S3:
+
+```java
+BatchSource<String> source = FileSources
+    .files("s3a://bucket-id/path/to/my/directory")
+    .build();
+```
+
+Read more in the [Programming
+Guide](https://jet-start.sh/docs/api/sources-sinks).
+
+## Enforce Strict Event Order
+
+Hazelcast Jet's primary focus is to leverage all opportunities to
+improve the throughput and latency of its computation. One example is
+using logic that isn't sensitive to the exact event order. Jet can use
+this freedom to optimally load-balance the data across parallel tasks.
+This works great for stateless transforms like `map` and `filter` as
+well as aggregate operations specifically written in terms of
+commutative and associative functions. However, Jet also supports
+transforms such as `mapStateful`, where reordering any two events is
+likely to result in different output.
+
+In version 4.4 we provide a new option,
+`pipeline.setPreserveOrder(true)`, which tells Jet to disable the
+dataflow optimizations that result in reordered events. One consequence
+of enabling it is that the level of parallelism in the source stage
+determines the parallelism of all the subsequent stages and the data
```

Comment:
Rather than saying "of all subsequent stages" we should say "of subsequent stages until it's repartitioned by using `rebalance()` or `groupingKey()`"